examples/versions.tf

```diff
@@ -7,7 +7,7 @@ terraform {
   required_providers {
     lightstep = {
       source  = "lightstep/lightstep"
-      version = "~> 1.61.1"
+      version = "~> 1.61.2"
```

Comment:
oops, I changed the wrong file to update the version number 🤦‍♀️ this was totally not it

Reply:
Here's the follow-up: https://github.com/lightstep/terraform-provider-lightstep/pull/190